content/blog/eclipse-temurin-slsa2/index.md

```diff
@@ -0,0 +1,116 @@
+  repository - starting at
+  [make-adopt-build-farm.sh](https://github.com/adoptium/temurin-build/blob/master/build-farm/make-adopt-build-farm.sh)
+
+- **Provenance - available**: We create an SBOM in OWASP CycloneDX format
+  along with our releases which contains all of the information about how
+  the builds are produced, which should allow someone to rebuild if required
+  for any reason.  This includes the full set of parameters which we use to
+  the makejdk_any_platform.sh script (invoked from
+  make-adopt-build-farm.sh), the source repository tags (`scmRef`) which was
+  used by the build process to produce the builds, the output from the
+  openjdk `configure` invocation and various other pieces of information. 
+  We are continually evolving the specific details which we include in the
+  SBOM.  If you want to join in the discussion on the content, you can find
+  the conversation in
+  [temurin-build#3013](https://github.com/adoptium/temurin-build/issues/3013)
+  or talk to us in the #secure-dev slack channel in the Adoptium workspace.
+
+## SLSA Level 2
+
+Level 2 adds in additional requirements to provide some tamper resistance of
+the build process, mostly in the areas of having all of our code version
+controlled which we do by storing anything related to producing our buids in
+git in our case and also by using a build service to produce and distribute
+those builds - in our case this is our Jenkins CI server. In our release
+cycles since July 2022 we have also provided PGP signature verification
+information along with the SHA checksums that we previously had, which was
+the final part of achieving SLSA level 2 compliance. The requirements here
+are:
+
+- **Source - Version controlled**. All of our source code is stored in GitHub in
+  the jdkXX repositories such as
+  [jdk17u](https://github.com/adoptium/jdk17u/), which we mirror from the
+  [openjdk project](https://github.com/openjdk). These
+  are version controlled and have tags for each release which we build from.
+  Contributors agree to the rules of the
+  [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)
+
+- **Build - Build service**. All of the build steps are run using our Jenkins
+  build service - currently at https://ci.adoptopenjdk.net which is used to
+  perform the builds, generate the SBOMs, and build the installers where
+  applicable. The output from the builds are then posted into GitHub release
+  repositories named as temurinXX-binaries (e.g.
+  [temurin17-binaries](https://github.com/adoptium/temurin17-binaries) and
+  also exposed via our API and web site.
+
+- **Provenance - Authenticated**. We provide GPG checksums that users can use to
+  verify that the downloads they receive have come from the Adoptium CI
+  system. The verification process is defined in
+  [this blog post](https://blog.adoptium.net/2022/07/gpg-signed-releases)
+  and most of the process in there is applicable to the SBOMs for the latest
```

Comment:
```
most of the process in there is applicable
```

What isn't applicable and why? That's the question that pops to mind after reading this statement.

Reply:
That statement will need to be changed as I'm looking at changing how they are distributed at the moment but to answer the specific question, the API calls to retrieve the data won't be as they are in those examples so they can't just be cut & pasted for this new scenario.

Reply:
Text of that section now adjusted (shortened!)